performance-comparisons/v11.10/results/report_0_postgresql_bounded_6500.md

```
@@ -0,0 +1,711 @@
+| PostStore.Get | p99 | 15ms | 17ms | 2ms | 13.02
+| ChannelStore.CreateInitialSidebarCategories | p99 | 49ms | 55ms | 6ms | 12.24
+| UserStore.UpdateLastLogin | p99 | 18ms | 20ms | 2ms | 11.12
+| TeamStore.SaveMember | p99 | 60ms | 65ms | 5ms | 8.31
+| PostStore.Save | p99 | 34ms | 36ms | 2ms | 5.81
+| UserStore.Count | p99 | 38ms | 40ms | 2ms | 5.28
+| ChannelStore.GetAllChannelMembersNotifyPropsForChannel | p99 | 82ms | 86ms | 4ms | 4.90
+| ChannelStore.SaveMember | p99 | 86ms | 89ms | 3ms | 3.47
+### Store times avg (improved):
+| | | Base | Actual | Delta | Delta % |
+| --- | --- | --- | --- | --- | --- |
+| ThreadStore.MarkAllAsReadByTeam | avg | 2ms | 0s | -2ms | -122.87
```

markdownlint-cli2 (0.23.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)